Lab 2.ipynb

```diff
@@ -0,0 +1,644 @@
+      "text/plain": [
+       "<Figure size 864x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ppmf = ppmf1d\n",
+    "\n",
+    "for j in np.arange(1,5):\n",
+    "    ppmf = np.convolve(ppmf1d,ppmf,'full')\n",
+    "\n",
+    "fig,ax = plt.subplots(1,2,figsize=(12,5))\n",
+    "\n",
+    "ax[0].bar(np.arange(0,len(ppmf)),ppmf,width =1)\n",
+    "ax[0].set_xlim(-.5,100)\n",
+    "ax[0].set_xlabel('Total Background Counts')\n",
+    "ax[0].set_ylabel('Probability');\n",
+    "\n",
+    "ax[1].bar(np.arange(0,len(ppmf)),ppmf,width =1)\n",
+    "#ax[1].set_xlim(-.5,50)\n",
+    "ax[1].set_xlabel('Total Background Counts')\n",
+    "ax[1].set_ylabel('Log Scale Probability')\n",
+    "ax[1].set_yscale('log');\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The graph of the log plot of the probability looks like that of a poisson distribution. The fact that it remained a Poisson distribution after just a few convolutions makes sense conceptually because any poisson distributed random variable that is  added to another poisson distributed variable should maintain their quality of being poisson except for when a large number of them are summed together because then the central limit theorem states that the pmf will approach a Guassian. This can be explained mathematically as $\\Large{\\sum pois(\\lambda_1) +pois(\\lambda_2) = \\sum pois(\\lambda_1 + \\lambda_2)}$\n",
+    "\n",
+    "$$ = \\sum \\left( \\frac{e^{-\\lambda_1}\\lambda_1^x}{x!} + \\frac{e^{-\\lambda_2}\\lambda_2^y}{y!} \\right)$$\n",
+    "\n",
+    "$$= e^{-(\\lambda_1 + \\lambda_2)}\\sum \\left( \\frac{\\lambda_1^x}{x!} + \\frac{\\lambda_2^y}{y!} \\right)$$\n",
+    "\n",
+    "$$ \\text{in our case x = y = k} $$\n",
+    "\n",
+    "$$\\text{New Poisson} =  \\frac{e^{-(\\lambda_1 + \\lambda_2)}(\\lambda_1 + \\lambda_2)^k}{k!}$$\n",
+    "\n",
+    "This new added Poison has the same function as the original Poisson $\\frac{e^{-\\lambda}\\lambda^k}{k!}$ except the means of the two poisson's are now summed together"
```

Comment:
Couple typos here: the first "Poisson" is missing an "s" and the last "Poisson" doesn't need the apostrophe